instrumentation/gopkg.in/olivere/elastic/splunkelastic/README.md

```diff
@@ -0,0 +1,11 @@
+# Splunk instrumentation for `gopkg.in/olivere/elastic`
+
+This package provides OpenTelemetry instrumentation for the
+[gopkg.in/olivere/elastic.v{3,7}](https://gopkg.in/olivere/elastic.v7)
```

Comment:
Shouldn't we also instrument `github.com/olivere/elastic` and `github.com/olivere/elastic/v7` packages as these are the "supported" Go modules defined by: https://github.com/olivere/elastic#releases

Reply:
I think mentioning it explicitly here is good enough.